account_invoice_view_payment/views/account_payment_view.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record id="view_account_payment_invoice_form" model="ir.ui.view">
+        <field name="name">account.payment.invoice.form</field>
+        <field name="model">account.payment</field>
+        <field name="inherit_id"
+               ref="account.view_account_payment_form"/>
+        <field name="arch" type="xml">
+            <button name="post" position="after">
+                <button name="post_and_open_payment" class="oe_highlight"
+                        string="Validate &amp; View Payment" type="object"
+                        groups="account.group_account_user"/>
```

Comment:
change groups to "account.group_account_invoice" since in v11 this group can access to payments from invoice anyway. Please apply everywhere else in this module.

Reply:
Updated.